package.json

```diff
@@ -29,8 +29,8 @@
   "homepage": "https://github.com/prezly/theme-kit-nextjs#readme",
   "scripts": {
     "clean": "rimraf build/ node_modules/",
-    "prebuild": "npm run clean && npm install",
```

Comment:
There is an issue, because it runs npm install before build and override all the linked packages, and because of that React complains about several versions

Reply:
You need to move these commands somewhere else (e.g. in the `prerelease` script), cause we did have issues with `build` and `node_modules` directories not being cleaned before building a release.